### PR TITLE
Allow commented out beancount commands in input file

### LINKED ISSUE #132: support beancount-only sections in input ledger
Similarly to the current support offered by `NoL2B begin`/`NoL2B end` to _exclude_ Ledger-CLI sections from `ledger2beancount` processing, it would be nice to have the ability to support Beancount-only sections in the input Ledger-CLI file.

A concrete use case of mine is interspersing Beancount event directives into my ledger.

To support this we can add two markers, e.g., `L2Bonly begin`/`L2Bonly end`, and require that all content in between them is commented with `;`, as it will not necessarily be Ledger-valid syntax. `ledger2beancount` will strip the markers, uncomment the lines in between, and emit them unmodified in Beancount output.

### PR DESCRIPTION
**Pull request by tbm**
Fixes #132

### CODE REVIEW

File: docs/manual.md [hunk 1]
```diff
@@ -616,6 +616,29 @@ and `$ignore_marker end`.  This syntax is also useful for ledger
...
+ledger2beancount, it is sometimes useful to put beancount-specific
+commands in the input file.  Of course, they may not be valid in ledger.
+Therefore, you can put a commented out line in the ledger input, mark it
+with the `$keep_marker` and ledger2beancount will uncomment the line and
+put it in the output.
+
+Given the input
+
+	; 2013-11-03 note Liabilities:CreditCard "Called about fraud" ; L2Bonly
+
```
Comment (zacchiro) [inline docs/manual.md]: we should either add "L2Bonly" as the default value for $keep_marker (my preference), or add here another documentation line like:
```
and the configuration value:

  keep_marker: L2Bonly
```

File: tests/ignore.ledger [hunk 2]
```diff
@@ -32,3 +34,12 @@ account Assets:Test2
...
     Equity:Opening-Balance
 
+; L2Bonly begin
+;2014-07-09 event "location" "Paris, France"
+; 2018-09-01 event "location" "Bologna, Italy"
+; L2Bonly end
+
+; 2013-11-03 note Liabilities:CreditCard "Called about fraud" ; L2Bonly
+# 2013-11-03 note Liabilities:CreditCard "Called about fraud" ; L2Bonly
+*2013-11-03 note Liabilities:CreditCard "Called about fraud" ; L2Bonly
```
Comment (zacchiro) [inline tests/ignore.ledger]: I'm shocked to learn that the last 2 examples work too, but it's Ledger's fault, not yours :-)

Comment (tbm) [reply tests/ignore.ledger]: https://www.ledger-cli.org/3.0/doc/ledger3.html#Commenting-on-your-Journal

File: docs/manual.md (hunk 1 above)
Comment (tbm) [reply docs/manual.md]: Yeah, I was wondering about that.

**Pull request merged**